packages/react-core/src/components/Dropdown/Dropdown.js

```diff
@@ -0,0 +1,90 @@
+import React, { Children, cloneElement } from 'react';
+import styles from '@patternfly/patternfly-next/components/Dropdown/styles.css';
+import { css } from '@patternfly/react-styles';
+import PropTypes from 'prop-types';
+import DropdownMenu from './DropdownMenu';
+
+export const DropdownPosition = {
+  right: 'right',
+  left: 'left'
+};
+
+export const DropdownDirection = {
+  up: 'up',
+  down: 'down'
+};
+
+const propTypes = {
```

Comment:
Can you add comments for each prop.  We need them for the docs.

Reply:
Great catch!